Lib/test/test_ensurepip.py

```diff
@@ -76,6 +76,12 @@ def setUp(self):
         patched_os.path = os.path
         self.os_environ = patched_os.environ = os.environ.copy()
 
+        # ensure the pip wheel exists
+        pip_filename = os.path.join(test.support.STDLIB_DIR, 'ensurepip', '_bundled',
+                                    f'pip-{ensurepip._PIP_VERSION}-py3-none-any.whl')
+        if not os.path.isfile(pip_filename):
```

Comment:
I feel like if it didn't exist, there should be some code to clean it up after testing so that the dummy file doesn't get forgotten on disk. Tests should avoid side effects...